mypy/server/astmerge.py

```diff
@@ -489,6 +489,7 @@ def visit_type_var(self, typ: TypeVarType) -> None:
     def visit_param_spec(self, typ: ParamSpecType) -> None:
         typ.upper_bound.accept(self)
         typ.default.accept(self)
+        typ.prefix.accept(self)
```

Comment:
Should `NodeReplaceVisitor` above do the same? More importantly, why isn't its `process_param_spec_def` actually ever called (as well as `process_type_var_tuple_def`)?

Reply:
This leads to #14872 where `process_param_spec_def` was added, but apparently not connected to anything? It should probably be...

Reply:
`ParamSpec` definitions cannot have prefixes, it is purely mypy-internal thing to represent `Concatenate` in the type system. Also I noticed that `process_param_spec_def()` and `process_type_var_tuple_def()` are not called anywhere. Probably they should be called from `visit_class_def()`. But this is definitely a topic for a separate issue/PR.

Reply:
Yep, I remember what a PSpec prefix means:) I just wasn't able to make sure that `process_param_spec_def` is actually *called* on definitions and not on other `ParamSpec` occurrences